### PR TITLE
refactor: Tidy up Operator Dashboard

### PR DESCRIPTION
**Pull request by bmtcril**
Closes: #380 

- Make names consistent
- Remove unusable or useless charts
- Add a "slowest ClickHouse queries" chart
- Remove the Course Success and Learner Engagement tabs which just had "coming soon" text

Before screenshots:

Screenshot 2023-10-03 at 3 26 32 PM
Screenshot 2023-10-03 at 3 26 11 PM

After screenshots:

Screenshot 2023-10-03 at 3 29 20 PM
Screenshot 2023-10-03 at 3 28 58 PM

### CODE REVIEW

**Comment by Ian2012** [comment]
@bmtcril can you leave screenshots of the changes?

**Comment by bmtcril** [comment]
@Ian2012 all set

**Comment by Ian2012** [comment]
@bmtcril the k8s jobs are failing. Can you move the translations file to a different folder, create a configmap for it, and attach it to superset?

**Review by Ian2012 — APPROVED** [review]

**Comment by bmtcril** [comment]
@Ian2012 probably not today, I'll have to take a look next week

File: tutoraspects/patches/kustomization-configmapgenerator [hunk 1]
```diff
@@ -68,6 +68,13 @@
     labels:
         app.kubernetes.io/name: superset
 
+- name: superset-localization
+  files:{% for file in "aspects/apps/superset/localization"|walk_templates %}
+  - plugins/{{ file }}{% endfor %}
+  options:
+    labels:
+        app.kubernetes.io/name: superset
```
Comment (Ian2012) [inline tutoraspects/patches/kustomization-configmapgenerator:76]: nit: update label `superset-localization`

**Comment by Ian2012** [comment]
@bmtcril can you upgrade the tutor version to `16.1.3`

More context here: https://discuss.openedx.org/t/missing-py2neo-package-causing-build-issues-in-all-edx-platform-releases/11371

File: tutoraspects/patches/kustomization-configmapgenerator (hunk 1 above)
Comment (bmtcril) [reply tutoraspects/patches/kustomization-configmapgenerator:76]: The others around it are labeled this way, should they be changed as well?

Comment (Ian2012) [reply tutoraspects/patches/kustomization-configmapgenerator:76]: Yes, they should

**Pull request merged**